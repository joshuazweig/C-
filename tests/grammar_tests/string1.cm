int main() {
    char *arr;

    arr = "Hello, I am a test string!";
}

