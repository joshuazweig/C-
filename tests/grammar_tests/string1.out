ACCEPT
[program:
  [decls:
    [decls:]
    [fdecl:
      [typ: INT]
      ID
      LPAREN
      [formals_opt:]
      RPAREN
      LBRACE
      [vdecl_list:
        [vdecl_list:]
        [vdecl: [typ: [typ: CHAR] STAR] ID SEMI]
      ]
      [stmt_list:
        [stmt_list:]
        [stmt: [expr: ID ASSIGN [expr: STRING]] SEMI]
      ]
      RBRACE
    ]
  ]
  EOF
]
